spyder/plugins/explorer/tests/test_explorer.py

```diff
@@ -46,5 +64,152 @@ def test_project_explorer(project_explorer):
     assert project_explorer
 
 
+def create_folders_files(file_paths, project_dir):
+    """Function to create folders and files to be used in test functions."""
+    list_paths = []
+    for item in file_paths:
+        if osp.splitext(item)[1]:
+            if osp.split(item)[0]:
+                dirs, fname = osp.split(item)
+                dirpath = osp.join(project_dir, dirs)
+                if not osp.exists(dirpath):
+                    os.makedirs(dirpath)
+                    item_path = osp.join(dirpath, fname)
+            else:
+                item_path = osp.join(project_dir, item)
+        else:
+            dirpath = osp.join(project_dir, item)
+            if not osp.exists(dirpath):
+                os.makedirs(dirpath)
+                item_path = dirpath
+        if item_path.endswith('script.py'):
+            with open(item_path, 'w') as fh:
+                fh.write('Python')
+        elif item_path.endswith('script1.py'):
+            with open(item_path, 'w') as fh:
+                fh.write('Spyder4')
+        elif item_path.endswith('script2.py'):
+            with open(item_path, 'w') as fh:
+                fh.write('Jan-2019')
+        elif item_path.endswith('text.txt'):
+            with open(item_path, 'w') as fh:
+                fh.write('Hello World')
+        list_paths.append(item_path)
+    return list_paths
+
+
+@pytest.mark.parametrize('file_paths',
+                         [['script.py'],
+                          ['script.py', 'script1.py', 'testdir/script2.py'],
+                          ['subdir/innerdir/text.txt', 'testdir']])
+@pytest.mark.parametrize('path_method', ['absolute', 'relative'])
+def test_copy_path(explorer_with_files, path_method, file_paths):
+    """Test copy absolute and relative paths."""
+    project, cb = explorer_with_files
+    file_paths = create_folders_files(file_paths, project.directory)
+    home_directory = project.explorer.treewidget.fsmodel.rootPath()
+    project.explorer.treewidget.copy_path(fnames=file_paths,
+                                          method=path_method)
+    cb_output = cb.text(mode=cb.Clipboard)
+    file_paths = [_fn.replace(os.sep, '/') for _fn in file_paths]
```

Comment:
This is the standard used original in explorer/widget, I tried to follow already existing names convention. Check [Here](https://github.com/spyder-ide/spyder/blob/c59255e1ab4422c7084efd55072527f30defceb0/spyder/plugins/explorer/widgets.py#L367)

Reply:
I see your point and appreciate you trying to follow existing convention where possible, and certainly don't mean to blame for another developer's questionable choice. Our decade-old codebase contains a lot of outdated, short-sighted and sometimes even downright awful semantic and style choices, so its not always easy to balance between consistency and correctness.

Keeping all that in mind, I think we still should go with the far more Pythonic and clear ``file_path``, for two reasons. First, this is an entirely separate file, and that of our tests, which doesn't necessarily need to be bound by a particular variable naming scheme in the function its testing. Second, that convention apparently arose from the work by our first developer nearly a decade ago when good Pythonic practice and modern standards were in their infancy. If it is some sort of "standard", at least in hindsight it is not really a good one to continue following due to its large potential for confusion with a private function (I was convinced it was for a non-trivial amount of time, searching around for the function definition) as well as being cryptic and unhelpful.

To note, the latter is also true of using ``fh`` for files (file handler?) but I didn't point those out since it was the former in this case, being named just like a private/temp function, that made this one so egregious.

Reply:
I believe it is resolved. Thanks..

Reply:
Thanks!